.github/workflows/package-component.yaml

```diff
@@ -0,0 +1,58 @@
+name: Package on Tag
+
+on:
+  push:
+    tags:
+      - '*/*/*/v*.*.*'
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Extract tag details
+        id: tag_info
+        run: |
+          TAG=${GITHUB_REF#refs/tags/}
```

Comment:
at this point we should also do a check to make sure that the tag refers to an actual component

Reply:
Done by being stricter with the tag match